### PR TITLE
Added welcome screen

### PR DESCRIPTION
**Pull request by andreadelrio**
To cover issue #189.

### CODE REVIEW

File: src/Upload.js [hunk 1]
```diff
@@ -147,10 +147,11 @@ class Upload extends React.Component {
 
   handleProgress(progress, textState, file) {
     this.setState({
-      started: true,
+      started: false,
```
Comment (davidascher) [inline src/Upload.js]: this seems unrelated.

File: src/components/CaseEditor/CaseEditor.js [hunk 2]
```diff
@@ -156,6 +156,7 @@ class _CaseEditor extends Component {
                     keyword picker
                     <p className="sub-heading">
                       Related Content
+                      {this.state.andrea}
```
Comment (davidascher) [inline src/components/CaseEditor/CaseEditor.js]: hmm.. =)

File: src/Upload.js [hunk 3]
```diff
@@ -147,10 +147,11 @@ class Upload extends React.Component {
...
       preview: file.preview,
       progress: progress
     });
+    this.props.onSelectLanguage(file.preview);
```
Comment (davidascher) [inline src/Upload.js]: also unrelated?

**Comment by andreadelrio** [comment]
@davidascher Hmm seems like there's something up with git and branches on my machine. I just removed unrelated lines.
